_data/events/melbourne.yaml

```diff
@@ -0,0 +1,26 @@
+event-info:
+  name: UNIHACK Melbourne 2018
+
+sponsors:
+- tier: Platnium Sponsor
+  type: col-md-4
+  companies: [telstra]
+
+- tier: Gold Sponsors
+  type: col-md-3 col-6
+  companies: [accenture, atlassian, bosch, imc, infosys, wade, xero]
+  missing: [australia-post, wisetech]
```

Comment:
Note: `missing` key is more internal notation about what sponsors aren't featured on our site and images need to be collected.